zhazam/Modules/Menu/MenuViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MenuViewController.swift
+//  zhazam
+//
+//  Created by Sanzhar Alim on 4/5/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class MenuViewController: UIViewController {
+    
+    @IBOutlet private var logoView: LogoView!
+    @IBOutlet private var titleButtons: [UIButton]!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        logoView.animate()
```

Comment:
Why did you decided to abstract animation logic inside of the `LogoView` here, but kept it inside of the controller in case of `LoadingViewController`?

Reply:
fixed